controllers.ts

```diff
@@ -1,6 +1,26 @@
 import { PelisCollection, Peli } from "./models";
 
 class PelisController {
-  constructor() {}
+  pelis:PelisCollection
+  constructor() {
+    this.pelis = new PelisCollection;
+  }
+
+  get(options:any){
+    let resultado:Promise<any>
+    if(options.id){
+      resultado = this.pelis.getById(options.id);
+    }else if(options.search){
+      resultado = this.pelis.search(options.search);
+    }else{ resultado = this.pelis.getAll().then((peliculas)=>{
+      return peliculas
+    })}
```

Comment:
Hola! Acá podrías solo hacer  `resultado = this.pelis.getAll();` como hiciste con los demás métodos. Igual un detalle, lo demás re bien!!